src/main/java/ch/uzh/ifi/hase/soprafs23/service/GameService.java

```diff
@@ -281,8 +283,14 @@ public void checkWinner(Game game) throws IOException, InterruptedException {
 
             // check if it is the guest or the host who won and set them as the winnre
             if (game.getGuestPoints() > game.getHostPoints()) {
+                // game shouldnt need to be saved and flushed to repo since we dont use it anymore
```

Comment:
@itsniezwe for debugging it's better to have it always saved, though I think that is covered in the parent method.